### PR TITLE
[SPARK-8282][SparkR] Make number of threads used in RBackend configurable

### PR DESCRIPTION
**Pull request by falaki**
Read number of threads for RBackend from configuration.

[SPARK-8282] #comment Linking with JIRA

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/api/r/RBackend.scala [hunk 1]
```diff
@@ -29,6 +29,7 @@ import io.netty.channel.socket.nio.NioServerSocketChannel
 import io.netty.handler.codec.LengthFieldBasedFrameDecoder
 import io.netty.handler.codec.bytes.{ByteArrayDecoder, ByteArrayEncoder}
 
+import org.apache.spark.deploy.SparkConf
```
Comment (andrewor14) [inline core/src/main/scala/org/apache/spark/api/r/RBackend.scala]: wrong package

**Comment by andrewor14** [comment]
other than the fact that it doesn't build, LGTM

**Comment by falaki** [comment]
Thanks. Is ```sparkr.md``` the right place to document this? Do we have a central place for all the configs?

**Comment by shivaram** [comment]
I think `configuration.md` might be the right place to document this

**Comment by shivaram** [comment]
Jenkins, retest this please

File: core/src/main/scala/org/apache/spark/api/r/RBackend.scala [hunk 2]
```diff
@@ -29,6 +29,7 @@ import io.netty.channel.socket.nio.NioServerSocketChannel
 import io.netty.handler.codec.LengthFieldBasedFrameDecoder
 import io.netty.handler.codec.bytes.{ByteArrayDecoder, ByteArrayEncoder}
 
+import org.apache.spark.SparkConf
 import org.apache.spark.Logging
```
Comment (andrewor14) [inline core/src/main/scala/org/apache/spark/api/r/RBackend.scala]: super minor, but these two imports can be merged. I can fix this when I merge this.

**Comment by andrewor14** [comment]
LGTM thanks I'm merging this into master and 1.4

**Pull request closed without merging**